addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -447,7 +447,13 @@ var FileWidget = SearchableMediaWidget.extend({
                 img.access_token = access_token[0];
             });
         }
-
+        return this._renderMedia(img);
+    },
+    /**
+     * @param {Object} img
+     * @returns
+     */
+    _renderMedia: function (img) {
```

Comment:
More default editor stuff.
Those should probably be extracted into separate commits to get their own localized explanation since they seem to be generic (probably just allowing more knowledge specific stuff later but it's hard to see).